### DISCUSSION TITLE
Problem with SMTP inputs

### DISCUSSION THREAD

**Discussion by codessensei** [factionsecurity/faction#31] [Q&A]
Hello. I encountered an error while trying to configure SMTP settings. Inputs include SMTP server, port, password and so on. When you type and save, a few TAB spaces are added before and after the text written to the input. I think this is why I couldn't get SMTP and e-mails to work. Does anyone know the solution to this problem?
![image](https://github.com/factionsecurity/faction/assets/66494203/0a263990-ece7-4182-ae2d-7468b652e2e9)

Edit : 

Also i can not select checklist from Checlists tab in Assesment page of project. I created checklists, but the dropdown does not appear in the list. 

![image](https://github.com/factionsecurity/faction/assets/66494203/1ac6048b-5afa-47fd-a51a-5e138c9effca)

Also i can not generate report. When i click to generate report button under finalize tab loader waiting for 3-4 second and not showing any error or any information about it. 

![image](https://github.com/factionsecurity/faction/assets/66494203/a9eeeaac-6e46-4b3d-b9ee-cba90f6d9111)

**Comment by summitt** [comment]
Closed as this is a dup of #33
